Don't compact URLs

SuperSandro2000 opened the pull request.

SuperSandro2000 committed:
Save rtree with go 1.20

majewsky reviewed (changes requested):
Compact URLs are an intentional part of the interface design. If we want to write the canonical URLs into `git remote`, that's fine by me. But user-facing URLs should stay compacted.

SuperSandro2000 committed:
Only use compact URL in UI, not in written files

many tools like eg some vscode extensions can parse the git remotes but do not respect
git aliases and fail to understand shorthands like gh:majewsky/gofu
